### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by kay0u**
[Build Status/badge/icon)](https://ci-apps-dev.yunohost.org/jenkins/job/minidlna_ynh%20PR18%20(Kay0u)/)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,9 +39,9 @@ Not relevant.
 
 #### Supported architectures
 
-* x86-64b - [![](https://ci-apps.yunohost.org/ci/logs/minidlna%20%28Community%29.svg)](https://ci-apps.yunohost.org/ci/apps/minidlna/)
-* ARMv8-A - [![](https://ci-apps-arm.yunohost.org/ci/logs/minidlna%20%28Community%29.svg)](https://ci-apps-arm.yunohost.org/ci/apps/minidlna/)
-* Jessie x86-64b - [![](https://ci-stretch.nohost.me/ci/logs/minidlna%20%28Community%29.svg)](https://ci-stretch.nohost.me/ci/apps/minidlna/)
+* x86-64b - [![](https://ci-apps.yunohost.org/ci/logs/minidlna%20%28Apps%29.svg)](https://ci-apps.yunohost.org/ci/apps/minidlna/)
+* ARMv8-A - [![](https://ci-apps-arm.yunohost.org/ci/logs/minidlna%20%28Apps%29.svg)](https://ci-apps-arm.yunohost.org/ci/apps/minidlna/)
+* Jessie x86-64b - [![](https://ci-stretch.nohost.me/ci/logs/minidlna%20%28Apps%29.svg)](https://ci-stretch.nohost.me/ci/apps/minidlna/)
```
Comment (ericgaspar) [inline README.md:44]:
```suggestion

```
Jessie is dead, long live Buster

File: README_fr.md [hunk 2]
```diff
@@ -39,9 +39,9 @@ Non applicable.
 
 #### Architectures supportées.
 
-* x86-64b - [![](https://ci-apps.yunohost.org/ci/logs/minidlna%20%28Community%29.svg)](https://ci-apps.yunohost.org/ci/apps/minidlna/)
-* ARMv8-A - [![](https://ci-apps-arm.yunohost.org/ci/logs/minidlna%20%28Community%29.svg)](https://ci-apps-arm.yunohost.org/ci/apps/minidlna/)
-* Jessie x86-64b - [![](https://ci-stretch.nohost.me/ci/logs/minidlna%20%28Community%29.svg)](https://ci-stretch.nohost.me/ci/apps/minidlna/)
+* x86-64b - [![](https://ci-apps.yunohost.org/ci/logs/minidlna%20%28Apps%29.svg)](https://ci-apps.yunohost.org/ci/apps/minidlna/)
+* ARMv8-A - [![](https://ci-apps-arm.yunohost.org/ci/logs/minidlna%20%28Apps%29.svg)](https://ci-apps-arm.yunohost.org/ci/apps/minidlna/)
+* Jessie x86-64b - [![](https://ci-stretch.nohost.me/ci/logs/minidlna%20%28Apps%29.svg)](https://ci-stretch.nohost.me/ci/apps/minidlna/)
```
Comment (ericgaspar) [inline README_fr.md:44]:
```suggestion

```

File: app_minidlna_fr.md [hunk 3]
```diff
@@ -7,9 +7,9 @@ Minidlna ne dispose pas d'une interface graphique, mais ne nécessite pas de con
 
 ### Quels fichiers multimédias sont partagés?
 Minidlna partage le dossier /home/yunohost.multimedia/share, qui est commun à chaque utilisateur dans le dossier /home/$USER/Multimedia/Share.
-[Plus d'informations sur les dossiers multimedia ici.](https://github.com/maniackcrudelis/yunohost.multimedia)
+[Plus d'informations sur les dossiers multimedia ici.](https://github.com/YunoHost-Apps/yunohost.multimedia)
 
-Si [transmission](https://github.com/Kloadut/transmission_ynh) est installé, les médias téléchargés seront disponible en dlna.
+Si [transmission](https://github.com/YunoHost-Apps/transmission_ynh) est installé, les médias téléchargés seront disponible en dlna.
 
 ### Comment consulter et lire les fichiers multimédias partagés par minidlna?
```
Comment (ericgaspar) [inline app_minidlna_fr.md:14]:
```suggestion
### Comment consulter et lire les fichiers multimédias partagés par MiniDLNA?
```

File: scripts/actions/install_backports [hunk 4]
```diff
@@ -26,24 +25,24 @@ app=${YNH_APP_INSTANCE_NAME:-$YNH_APP_ID}
...
+	ynh_die --message="You are already using the version from backports repository." --ret_code=0
 fi
 
 #=================================================
 # SPECIFIC ACTION
 #=================================================
 # RE-INSTALL MINIDLNA FROM BACKPORTS
 #=================================================
-ynh_script_progression --message="Re-install minidlna from backports" --weight=9
+ynh_script_progression --message="Re-installing minidlna from backports..." --weight=9
```
Comment (ericgaspar) [inline scripts/actions/install_backports:36]:
```suggestion
ynh_script_progression --message="Re-installing MiniDLNA from backports..." --weight=9
```

File: scripts/actions/install_standard [hunk 5]
```diff
@@ -23,19 +23,19 @@ source /usr/share/yunohost/helpers
...
+	ynh_die --message="You are already using the version from the stable repository." --ret_code=0
 fi
 
 #=================================================
 # SPECIFIC ACTION
 #=================================================
 # RE-INSTALL MINIDLNA FROM STABLE
 #=================================================
-ynh_script_progression --message="Re-install minidlna from stable" --weight=9
+ynh_script_progression --message="Re-installing minidlna from stable..." --weight=9
```
Comment (ericgaspar) [inline scripts/actions/install_standard:34]:
```suggestion
ynh_script_progression --message="Re-installing MiniDLNA from stable..." --weight=9
```

File: scripts/backup [hunk 6]
```diff
@@ -33,26 +33,26 @@ version=$(ynh_app_setting_get $app version)
...
+ynh_script_progression --message="Backing up inotify's config..."
 
-ynh_backup "/etc/sysctl.d/90-inotify_minidlna.conf"
+ynh_backup --src_path="/etc/sysctl.d/90-inotify_minidlna.conf"
 
 #=================================================
 # BACKUP OF MINIDLNA CONFIGURATION
 #=================================================
-ynh_script_progression --message="Backup of minidlna configuration"
+ynh_script_progression --message="Backing up minidlna configuration..."
```
Comment (ericgaspar) [inline scripts/backup:50]:
```suggestion
ynh_script_progression --message="Backing up MiniDLNA configuration..."
```

File: scripts/install [hunk 7]
```diff
@@ -70,12 +70,12 @@ ynh_multimedia_build_main_dir
 #=================================================
 # INSTALL MINIDLNA
 #=================================================
-ynh_script_progression --message="Install minidlna" --weight=45
+ynh_script_progression --message="Installing minidlna..." --weight=45
```
Comment (ericgaspar) [inline scripts/install:73]:
```suggestion
ynh_script_progression --message="Installing MiniDLNA..." --weight=45
```

File: scripts/remove [hunk 8]
```diff
@@ -67,4 +67,4 @@ fi
 # END OF SCRIPT
 #=================================================
 
-ynh_script_progression --message="Deletion completed" --last
+ynh_script_progression --message="Removal of $app completed" --last
```
Comment (ericgaspar) [inline scripts/remove:70]:
```suggestion
ynh_script_progression --message="Removal of MiniDLNA completed" --last
```

File: scripts/restore [hunk 9]
```diff
@@ -40,21 +40,21 @@ ynh_exec_fully_quiet yunohost firewall allow --no-upnp UDP 1900
...
-ynh_script_progression --message="Create yunohost.multimedia directory" --weight=6
+ynh_script_progression --message="Creating yunohost.multimedia directory..." --weight=6
 
 ynh_multimedia_build_main_dir
 
 #=================================================
 # INSTALL MINIDLNA
 #=================================================
-ynh_script_progression --message="Install minidlna" --weight=45
+ynh_script_progression --message="Installing minidlna" --weight=45
```
Comment (ericgaspar) [inline scripts/restore:50]:
```suggestion
ynh_script_progression --message="Installing MiniDLNA" --weight=45
```

File: app_minidlna.md [hunk 10]
```diff
@@ -6,18 +6,18 @@ It allows to easily share multimedia files with any compatible devices present o
 Minidlna does not have a graphical interface, but does not require any special configuration.
```
Comment (ericgaspar) [inline app_minidlna.md:6]:
```suggestion
MiniDLNA does not have a graphical interface, but does not require any special configuration.
```

File: app_minidlna.md [hunk 11]
```diff
@@ -6,18 +6,18 @@ It allows to easily share multimedia files with any compatible devices present o
...
 
 ### What multimedia files are shared?
-Minidlna sharing the folder /home/yunohost.multimedia/share, which is common to each user in /home/$USER/Multimedia/Share.
-[More information about multimedia files here.](Https://github.com/maniackcrudelis/yunohost.multimedia)
+Minidlna is sharing the folder /home/yunohost.multimedia/share, which is common to each user in /home/$USER/Multimedia/Share.
```
Comment (ericgaspar) [inline app_minidlna.md:9]:
```suggestion
MiniDLNA is sharing the folder `/home/yunohost.multimedia/share`, which is common to each user in `/home/$USER/Multimedia/Share`.
```

File: app_minidlna.md [hunk 12]
```diff
@@ -6,18 +6,18 @@ It allows to easily share multimedia files with any compatible devices present o
...
+[More information about multimedia files here.](https://github.com/YunoHost-Apps/yunohost.multimedia)
 
-If [transmission](https://github.com/Kloadut/transmission_ynh) is installed, the downloaded media will be available in dlna.
+If [transmission](https://github.com/YunoHost-Apps/transmission_ynh) is installed, the downloaded media will be available in dlna.
```
Comment (ericgaspar) [inline app_minidlna.md:12]:
```suggestion
If [Transmission](https://github.com/YunoHost-Apps/transmission_ynh) is installed, the downloaded media will be available in DLNA.
```

File: app_minidlna.md [hunk 13]
```diff
@@ -6,18 +6,18 @@ It allows to easily share multimedia files with any compatible devices present o
...
 
 ### How to view and play media files shared by minidlna?
```
Comment (ericgaspar) [inline app_minidlna.md:14]:
```suggestion
### How to view and play media files shared by MiniDLNA?
```

File: app_minidlna.md [hunk 14]
```diff
@@ -6,18 +6,18 @@ It allows to easily share multimedia files with any compatible devices present o
...
 To view and play media files, all you need is a compatible client DLNA/UPNP.
 
 The majority of set-top boxes provided by ISPs are DLNA compatible, simply look for sources of external media.
 This is also true for the latest generation game consoles connected to internet.
 
-Some TV and blu-ray player is also DLNA compatible.
+Some TV and blu-ray player are also DLNA compatible.
 
 In any case, it is generally sufficient to seek external sources, USB etc, to find the DLNA server, displayed under the name **Yunohost DLNA**.
```
Comment (ericgaspar) [inline app_minidlna.md:22]:
```suggestion
In any case, it is generally sufficient to seek external sources, USB etc., to find the DLNA server, displayed under the name **YunoHost DLNA**.
```

File: app_minidlna_fr.md [hunk 15]
```diff
@@ -7,9 +7,9 @@ Minidlna ne dispose pas d'une interface graphique, mais ne nécessite pas de con
 
 ### Quels fichiers multimédias sont partagés?
 Minidlna partage le dossier /home/yunohost.multimedia/share, qui est commun à chaque utilisateur dans le dossier /home/$USER/Multimedia/Share.
```
Comment (ericgaspar) [inline app_minidlna_fr.md:9]:
```suggestion
MiniDLNA partage le dossier `/home/yunohost.multimedia/share`, qui est commun à chaque utilisateur dans le dossier `/home/$USER/Multimedia/Share`.
```

**Review by ericgaspar — CHANGES_REQUESTED** [review]
Some typo stuff. I should probably push a PR for that...

**Review by ericgaspar — APPROVED** [review]

File: scripts/upgrade [hunk 16]
```diff
@@ -94,12 +94,13 @@ fi
...
 # INCREASE INOTIFY'S LIMITS
 #=================================================
-ynh_script_progression --message="Increase inotify's limits" --weight=2
+ynh_script_progression --message="Increasing inotify's limits..." --weight=2
 
 # Increase the maximum number of files inotify can monitor.
 cp -a ../conf/90-inotify_minidlna.conf /etc/sysctl.d/
 # Then, reload the kernel configuration.
-if ! IS_PACKAGE_CHECK; then
+if ! IS_PACKAGE_CHECK   # LXC doesn't allow sysctl to play with kernel options.
```
Comment (alexAubin) [inline scripts/upgrade:102]: Dunno if that's relevant but we had a discussion in transmission a while ago about the usage of this technique ... sure that fixes tests, but that also means that the app won't install on regular LXC instances (for example, our very own infra is based on yunohost LXCs ...) which are a legit use case. One could use systemctl-detect-virt to detect that we're inside a lxc (or idk which trick we ended up using in transmission)

Anyway, I understand that the point of the PR was only to changes the syntax + add the comment :/

**Review by alexAubin — APPROVED** [review]
LGTM apart from a few minor comments

File: scripts/_common.sh [hunk 17]
```diff
@@ -730,128 +370,381 @@ ynh_maintenance_mode_OFF () {
...
+# Set as 'medium', the process manager will be at dynamic. If the service is idle, a number of children
+#   equal to pm.min_spare_servers will stay alive. So the service can be quick to answer to any request.
+#   The number of children can grow if needed. The footprint can stay low if the service is idle, but
+#   not null. The impact on the proc is a little bit less than 'ondemand' as there's always a few
+#   children already available.
+# Set as 'high', the process manager will be set at 'static'. There will be always as many children as
+#   'pm.max_children', the footprint is important (but will be set as maximum a quarter of the maximum
+#   RAM) but the impact on the proc is lower. The service will be quick to answer as there's always many
+#   children ready to answer.
+ynh_get_scalable_phpfpm () {
```
[the remark this answers is not in the document]
Comment (kay0u) [reply scripts/_common.sh]: You're right, moreover, these helpers are not used.

File: scripts/install [hunk 18]
```diff
@@ -19,49 +19,49 @@ ynh_abort_if_errors
...
 
 #=================================================
 # CHECK DEBIAN'S CODENAME
 #=================================================
-ynh_script_progression --message="Check debian's codename"
+ynh_script_progression --message="Checking Debian's codename..."
 
 codename=$(ynh_debian_release)
-test -z "$codename" && (ynh_die "codename empty")
+test -z "$codename" && (ynh_die --message="codename empty")
```
[the remark this answers is not in the document]
Comment (kay0u) [reply scripts/install:39]: True, but I've remove the backport list now :-)

**Review by ericgaspar — APPROVED** [review]

**Pull request merged**